demo/sidebars.js

```diff
@@ -89,7 +89,19 @@ module.exports = {
           dirName: "options/miscellaneous"
         }
       ]
-    }
+    },
+    {
+      type: "category",
+      label: "PostProcessing",
```

Comment:
How about changing the name to Post-processing?

Reply:
I'll do it :)